src/actions/index.js

```diff
@@ -0,0 +1,5 @@
+export const addImages = (data) => ({type: 'ADD', data: data})
+export const deleteImage = () => ({type: 'DELETE'})
+export const updateCountToDelete = (countToDelete) => ({type: 'UPDATE_COUNT', countToDelete: countToDelete})
+export const startDowloadingImages = () => ({type: 'START'})
+export const endDowloadingImages = () => ({type: 'END'})
```

Comment:
Since you extracted the action types to separate constants, why not re-use them everywhere?

Reply:
Resolved.